src/sock.c

```diff
@@ -95,13 +98,8 @@ int opensock (const char *host, int port, const char *bind_to)
 
         n = getaddrinfo (host, portstr, &hints, &res);
         if (n != 0) {
-                const char *error;
-                if (n == EAI_SYSTEM)
-                        error = strerror (errno);
-                else
-                        error = gai_strerror (n);
                 log_message (LOG_ERR,
-                             "opensock: Could not retrieve address info for %s:%d:", host, port, error);
+                             "opensock: Could not retrieve address info for %s:%d: ", host, port, get_gai_error (n));
```

Comment:
wouldnt we need an additional %s here ?

Reply:
Of course. Was missing since my first commit :(